config/config.exs

```diff
@@ -9,6 +9,9 @@ import Config
 
 config :elixir, :time_zone_database, Tzdata.TimeZoneDatabase
 
+config :bike_brigade, BikeBrigade.LocalizedDateTime,
+  timezone: System.get_env("APP_TZ") || "America/Toronto"
```

Comment:
```suggestion
  timezone: System.get_env("APP_TZ", "America/Toronto")
```

Worth nothing that `System.get_env` takes a second optional argument for default